lib/compat/wordpress-6.1/class-wp-theme-json-6-1.php

```diff
@@ -1164,7 +1164,7 @@ public function set_spacing_sizes() {
 			return null;
 		}
 
-		$unit            = sanitize_title( $spacing_scale['unit'] );
+		$unit            = '%' === $spacing_scale['unit'] ? $spacing_scale['unit'] : sanitize_title( $spacing_scale['unit'] );
```

Comment:
```suggestion
		$unit            = '%' === $spacing_scale['unit'] ? '%' : sanitize_title( $spacing_scale['unit'] );
```
I can't see that it makes a difference, but this feels slightly safer.

Reply:
yeh, slightly more succinct as well